examples/app-npm-webpack/webpack-config.js

```diff
@@ -22,19 +27,33 @@ module.exports = {
 			"globalize/dist/globalize-runtime/unit"
 		]
 	} : "./app/index.js",
-	debug: !options.production,
 	output: {
-		path: options.production ? "./dist" : "./tmp",
+		path: path.resolve(__dirname, options.production ? "./dist" : "./tmp"),
 		publicPath: options.production ? "" : "http://localhost:8080/",
-		filename: options.production ? "app.[hash].js" : "app.js"
+		filename: options.production ? "[name].[chunkhash].js" : "app.js"
 	},
 	resolve: {
-		extensions: [ "", ".js" ]
+		extensions: [ "*", ".js" ]
 	},
 	plugins: [
 		new HtmlWebpackPlugin({
 			production: options.production,
-			template: "./index-template.html"
+			template: "./index-template.html",
+
+			// FIXME: These 2 options are a workaround to insert i18n/[locale].[hash].js files at right spot just before main.[hash].js
+			chunks: options.chunks,
+			chunksSortMode: function (chunk1, chunk2) {
+				const order1 = options.chunks.indexOf(chunk1.names[0]);
+				const order2 = options.chunks.indexOf(chunk2.names[0]);
+
+				if (order1 > order2) {
+					return 1;
+				}
+				if (order1 < order2) {
+					return -1;
+				}
+				return 0;
+			}
```

Comment:
Does it need to be fixed in the webpack plugin instead?

Reply:
I believe this is here as a better solution than doing something similar in the template, right? Although, it would add all the locales `i18n/[locale].[hash].js`. The template's purpose is just to give users a basic idea of what they need to do. Users are probably using a better rendering approach in their apps anyway. :)

Reply:
Your description has a good summary about this, thanks.

> However if you keep the i18n/en dependency and just add another one below, it seems to work as the latter de overrules the former en:

Yeap, this shouldn't happen (i.e., it shouldn't be necessary to keep en too). The test I'd make was to keep template as basic and simple as it was, i.e., to make minimal updates in this example necessary to support webpack 2.

Reply:
I hope the ordering issue can be solved within the plugin without the need to specify the chunks and ordering here.